tracer/src/Datadog.Trace/ExtensionMethods/DictionaryExtensions.cs

```diff
@@ -82,5 +82,11 @@ public static bool TryGetValue<TValue>(this IDictionary dictionary, object key,
         }
 
         public static bool IsNullOrEmpty<TKey, TValue>(this IReadOnlyDictionary<TKey, TValue> dictionary) => dictionary?.IsEmpty() ?? true;
+
+        public static void Deconstruct<T1, T2>(this KeyValuePair<T1, T2> tuple, out T1 key, out T2 value)
```

Comment:
This already exists in netcore2.1+ right? Needs an `#if`?

Reply:
Actually, it not in use, so I've removed it in 394326947ac2b415ca04f4c65fb9d7916a47e10e